docs/docs/typography-js.md

```diff
@@ -2,7 +2,87 @@
 title: Typography.js
 ---
 
-This is a stub. Help our community expand it.
+# Using Typography.js in Gatsby
+Typography.js is a javascript library that enables you to define and explore the typographic design of your website and define beautiful custom and pre-existing typographic themes. It limits the number of tedious changes you need to make to your website just to change the font. Typography.js currently maintains over 30 themes for you to use, however you can also define your own custom font themes if none of the available themes meet your requirements. Implementing Typography into your project involves specifying a configuration object for Typography and installing a Gatsby plugin.
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+## Installing the Typography plugin
+Gatsby has the plugin `gatsby-plugin-typography` to assist with introducting Typography.js library into your project.
+
+You can install the plugin into your project by running the command `npm install gatsby-plugin-typography --save`
+
+After the installation of the plugin has completed, navigate to your `gatsby-config.js` file located in the root of your project's directory and add the plugin to the configuration:
+
+```diff
+module.exports = {
+siteMetadata: {
+    title: 'Gatsby Default Starter',
+},
+plugins: [
++ {
++  resolve: `gatsby-plugin-typography`,
++  options: {
++    pathToConfigModule: `src/utils/typography`,
++  }
++ }
+],
+}
+```
+
+`gatsby-plugin-typography` takes two options for you to specify:
+
+ - **pathToConfigModule**: (string) The path to the file in which you export your typography configuration.
+ -  **omitGoogleFont**: (boolean, default: false) Typography includes a helper that makes a request to Google’s font CDN for the fonts you
+   need. You might, however, want to inject the fonts into JS or use a
+   CDN of your choosing. Setting this value to true will make
+   gatsby-plugin-typography skip the inclusion of this helper. You will
+   have to include the appropriate fonts yourself.
+
+
+## Creating the Typography configuration
+
+Now that you have added the plugin, create the directory `src/utils/` if it does not already exist in your project and add a new file name `typography.js`. This file will be used to specify the typography configuration and used as the path for the `pathToConfigModule` option that the plugin requires.
+
+Inside the `typography.js` file you just created, you can now define your websites typography configuration. A basic typography.js configuration looks like this:
+
+```js
+import Typography  from  "typography";
```

Comment:
The spacing here is a bit wonky, right?

`import Typography  from  "typography"`

we should just use single space for all things, and not tab indents or anything!

Other than that, this looks great.

Reply:
@DSchau All done. Hope that's better, let me know otherwise should be good for merge. Will revise this more when I get the opportunity, running low on time atm 👍